packages/@react-aria/landmark/test/useLandmark.test.tsx

```diff
@@ -70,21 +70,23 @@ function toggleBrowserWindow() {
 
 
 describe('LandmarkManager', function () {
-  let offsetWidth, offsetHeight;
+  let offsetWidth, offsetHeight, spyWarn;
 
   beforeAll(function () {
     offsetWidth = jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 1000);
     offsetHeight = jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 1000);
+    spyWarn = jest.spyOn(console, 'warn').mockImplementation(() => {});
```

Comment:
lets only mock the console for the test in question
I want all other console warnings to come through so the tests fail if we encounter unexpected ones